kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -522,4 +522,15 @@ object KyuubiConf {
     .transform(_.toUpperCase(Locale.ROOT))
     .checkValues(ShareLevel.values.map(_.toString))
     .createWithDefault(ShareLevel.USER.toString)
+
+  /////////////////////////////////////////////////////////////////////////////////////////////////
+  //                                   Engine Configuration                                      //
+  /////////////////////////////////////////////////////////////////////////////////////////////////
+
+  val ENGINE_INITIALIZE_SQL: ConfigEntry[String] = buildConf("engine.initialize.sql")
+    .doc("SemiColon-separated list of SQL statements to be initialized in the newly created " +
+      "engine before queries. By default, it is `SELECT CURRENT_DATABASE()`.")
+    .version("1.2.0")
+    .stringConf
+    .createWithDefault("SELECT CURRENT_DATABASE()")
```

Comment:
Let's change it back to `show databases`, and create a specific test suite to add some tests for `SELECT CURRENT_DATABASE();` and etc